server/dialer/dial.go

```diff
@@ -39,6 +42,12 @@ type UserData struct {
 	Credential pwshow.UserList `json:"Credential"`
 }
 
+//FileUploaded struct holds informations about how success is the process of uploading credential file
+type FileUploaded struct {
+	Lines   []int `json:"Lines"`
```

Comment:
what is `Lines` for ?

Reply:
I need that as information to give to the user ,if a particular line in the uploaded csv file contains a credentials that already exist in the database . all that to avoid repetition .